it/test/AlcoholDutyReturnsConnectorISpec.scala

```
@@ -105,6 +105,32 @@ class AlcoholDutyReturnsConnectorISpec extends ISpecBase with WireMockHelper{
         }
       }
 
+      "should fail when a return is submitted, but something other than a CREATED (202) Status code is returned" in new SetUp {
```

Suggest rename test to... 

"_should fail when an unexpected status code is returned_" OR;

"_submission should fail when an unexpected status code is returned_"